### ISSUE TITLE
Add sensors

### ISSUE THREAD

**Issue by andyblarblar** [iscumd/KiloOhm#2]
Add driver nodes for GPS, Encoders (needs to be done on rtq repo), IMU, and Camera.

**Comment by andyblarblar** [comment]
https://github.com/dawonn/vectornav/tree/ros2 Looks like a good GPS/IMU

https://github.com/klintan/ros2_usb_camera Looks like a good start for the Flir

**Comment by girvenavery2022** [comment]
so the Ouster OS1 has a built in internal IMU that is actually very good, so we should be able to get away with using that as our IMU for Ohm

**Comment by andyblarblar** [comment]
it looks like the gps reliably mounts on ttyUSB0, with a vendor id of FTDI. This doesnt race condition with any other device, so we should be good leaving it hard coded.

**Comment by girvenavery2022** [comment]
yeah as long as we don't add a usb device that mounts to ttyUSB[0-9] we should be good. Luckily Rboteq mounts to ttyACM

**Comment by andyblarblar** [comment]
Update:
- GPS connects, but we cannot get it to publish
- Encoder connects, but we have an issue with it blocking the roboteq drive commands
- Sick doesnt have an IMU

Because of these issues, Im planning on using the realsense tracking cam for odom until we get nav done. This should work good enough for now. I would still like to get the EKF working to integrate gps and encoders later, but as of now both seem to be up in the air.

**Comment by andyblarblar** [comment]
Update 2:

Encoders now work, but are not integrated. The plan is to merge the encoders into an odom message, then merge that odom message with the odom from the realsense in an ekf.

**Comment by andyblarblar** [comment]
So it looks like we have a very old version of the vectornav firmware, which is why it wont conncect. We need to get the new one direct from vectornav. This should mean we can get the GPS working however.
